ui-v2/app/templates/dc/services/show.hbs

```diff
@@ -49,12 +50,13 @@
             <h2>Healthy Nodes</h2>
             {{#list-collection
                 items=healthy
-                cell-layout=(percentage-columns-layout healthy.length columns 100) as |item index|
+                cell-layout=(percentage-columns-layout healthy.length columns 113) as |item index|
```

Comment:
Another peculiar number.

Reply:
Yup, so similar to the above its to do with maintaining the 22px margin, you have to specify a row height for the DOM recycling scrollers, and as the boxes grew to make room for the Server.ID this number needed to grow also. Would be nice to auto calculate I suppose, but its a bit 'chicken and egg' 🐔 🥚 (gotta love an emoji 😄 )